src/OpenTelemetry.Api/Trace/TracerProvider.cs

```diff
@@ -14,22 +14,40 @@
 // limitations under the License.
 // </copyright>
 
+using System;
 using System.Diagnostics;
 
 namespace OpenTelemetry.Trace
 {
     /// <summary>
-    /// TracerProvider is the entry point of the OTel API. It provides access to Tracers.
+    /// TracerProvider is the entry point of the OpenTelemetry API. It provides access to Tracers.
     /// </summary>
-    public abstract class TracerProvider
+    public class TracerProvider : IDisposable
     {
+        /// <summary>
+        /// Gets the default Tracer.
+        /// </summary>
+        public static TracerProvider Default = new TracerProvider();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TracerProvider"/> class.
+        /// </summary>
+        protected TracerProvider()
+        {
+        }
+
+        /// <inheritdoc/>
+        public virtual void Dispose()
+        {
+        }
+
         /// <summary>
         /// Gets a tracer with given name and version.
         /// </summary>
         /// <param name="name">Name identifying the instrumentation library.</param>
         /// <param name="version">Version of the instrumentation library.</param>
         /// <returns>Tracer instance.</returns>
-        public static Tracer GetTracer(string name, string version = null)
+        public Tracer GetTracer(string name, string version = null)
```

Comment:
Each Tracer news up an ActivitySource so users will also have to add that to their list of captured sources when they enable SDK? That's a little different from what we have today. Today if you do TracerFactoryBase.Default.GetTracer (something like that) and emit spans through that Tracer, they are automatically included. Not necessarily a problem, but I think it's a big change we should consider.

Reply:
Yes its a bit less user friendly, as one has to remember to include the ActivitySource to the list of sources to listen.
We need to find alternate and better ways to this problem. Maybe change listening model to be "listen to all, except those are explicitly excluded" or "listen to all sources starting with MyCompany.MyProduct.*" etc.
https://github.com/open-telemetry/opentelemetry-dotnet/issues/736